File: lib/ruby_app/elements/inputs/.cache/email_input.js
// /Users/fficnar/Dropbox/Projects/gems/ruby_app/lib/ruby_app/elements/inputs/email_input.js.haml
$(document).ready(function() {
  $('input[type="email"]').live('change', function(event) {
    event.preventDefault();
    RubyApp.queueEvent({_class:'RubyApp::Elements::Inputs::EmailInput::ChangedEvent', source_id:$(this).attr('id'), value:$(this).val()}, false);
  });
});

File: lib/ruby_app/elements/mobile/inputs/email_input.rb
module RubyApp

  module Elements

    module Mobile

      module Inputs
        require 'ruby_app/elements/mobile/input'

        class EmailInput < RubyApp::Elements::Mobile::Input

          template_path(:all, File.dirname(__FILE__))

          def initialize
            super
            self.attributes.merge!(:type => :email)
          end

        end

      end

    end

  end

end
